pdc_client/__init__.py

```diff
@@ -120,28 +119,19 @@ class to perform requests.
             except KeyError:
                 print("'%s' must be specified in configuration file." % CONFIG_URL_KEY_NAME)
                 sys.exit(1)
-            insecure = config.get(CONFIG_INSECURE_KEY_NAME, insecure)
+            ssl_verify = config.get(CONFIG_SSL_VERIFY_KEY_NAME, ssl_verify)
             develop = config.get(CONFIG_DEVELOP_KEY_NAME, develop)
             token = config.get(CONFIG_TOKEN_KEY_NAME, token)
 
+        self.session.verify = ssl_verify
+
         if not develop:
             # For local environment, we don't need to require a token,
             # just access API directly.
             # REQUIRED, OPTIONAL, DISABLED
             self.session.auth = requests_kerberos.HTTPKerberosAuth(
                 mutual_authentication=requests_kerberos.DISABLED)
 
-        if insecure:
-            # turn off for servers with insecure certificates
-            self.session.verify = False
-            # turn off warnings about making insecure calls
-            if [int(x) for x in requests.__version__.split('.')] < [2, 4, 0]:
```

Comment:
I think it's up to each application to decide what to do about the warnings. It's not behavior I expect from a library. If you want, I will add this functionality back to the scripts in `bin`.